dxl_joint_id_map:
    head_pitch: 20
    head_yaw: 21

    left_shoulder_swing_joint: 1
    left_shoulder_lateral_joint: 3
    left_elbow_joint: 5

    right_shoulder_swing_joint: 6
    right_shoulder_lateral_joint: 2
    right_elbow_joint: 4
